.github/workflows/ci.yml

```diff
@@ -243,7 +243,7 @@ jobs:
             exit 1
           fi
 
-  run-experimental-evals:
```

Comment:
can we have `run-text-extract-evals` run right after extract evals? we could also have two separate extract CI, one `run-text-extract` and one `run-dom-extract`. the way it's currently setup, CI to skip running text extract evals if combo evals fail. we can also add text extract as a param for combo/evals, but priority is future PRs shouldn't break text extract or dom extract.